Sources/Public/Animation/LottieView.swift

```diff
@@ -99,17 +76,102 @@ public struct LottieView: UIViewConfiguringSwiftUIView {
     }
   }
 
+  /// Returns a copy of this view with its accessibility label updated to the given value.
+  public func accessibilityLabel(_ accessibilityLabel: String?) -> Self {
+    configure { view in
+      #if os(macOS)
+      view.setAccessibilityElement(accessibilityLabel != nil)
+      view.setAccessibilityLabel(accessibilityLabel)
+      #else
+      view.isAccessibilityElement = accessibilityLabel != nil
+      view.accessibilityLabel = accessibilityLabel
+      #endif
+    }
+  }
+
+  /// Returns a copt of this view with its `LottieConfiguration` updated to the given value.
+  public func configuration(_ configuration: LottieConfiguration) -> Self {
+    var copy = self
+    copy.configuration = configuration
+
+    copy = copy.configure { view in
+      if view.configuration != configuration {
+        view.configuration = configuration
+      }
+    }
+
+    return copy
+  }
+
+  /// Returns a copy of this view with its image provider updated to the given value.
+  /// The image provider must be `Equatable` to avoid unnecessary state updates / re-renders.
+  public func imageProvider<ImageProvider: AnimationImageProvider & Equatable>(_ imageProvider: ImageProvider) -> Self {
+    var copy = self
+    copy.imageProvider = imageProvider
+
+    copy = copy.configure { view in
+      if (view.imageProvider as? ImageProvider) != imageProvider {
+        view.imageProvider = imageProvider
```

Comment:
Pretty neat -- in the example app where we're using `.imageProvider(.exampleAppSampleImages)`, this line here actually never runs. The correct image provider is used when initializing the `LottieAnimationView` above in `LottieAnimationView.swiftUIView { ... }` and then this doesn't need to run again since the equality check is succeeding.